test/unit/org/apache/cassandra/cql3/validation/operations/InsertTest.java

```diff
@@ -75,7 +75,7 @@ public void testEmptyTTL() throws Throwable
         createTable("CREATE TABLE %s (k int PRIMARY KEY, v int)");
         execute("INSERT INTO %s (k, v) VALUES (0, 0) USING TTL ?", (Object) null);
         execute("INSERT INTO %s (k, v) VALUES (1, 1) USING TTL ?", ByteBuffer.wrap(new byte[0]));
-        assertRows(execute("SELECT k, v, ttl(v) FROM %s"), row(1, 1, null), row(0, 0, null));
+        assertRowsIgnoringOrder(execute("SELECT k, v, ttl(v) FROM %s"), row(1, 1, null), row(0, 0, null));
```

Comment:
different partitioners produce different partition key orderings, so `IgnoringOrder` is required